findoutlie/data_load.py

```diff
@@ -92,3 +92,29 @@ def load_sub_run(sub_id, run_num, **kwargs):
         images = images[0]
 
     return images
+
+def load_image(fname):
```

Comment:
am I understanding correctly that the goal of this function is to avoid import nibabel everywhere we load images?

Reply:
That is correct indeed.